tests/test_diffeq/test_odefiltsmooth/test_ivpfiltsmooth.py

```diff
@@ -0,0 +1,109 @@
+"""Test the ODE filter on a more granular level.
+
+The tests in here go through probsolve_ivp(), but do not directly test
+the interface, but the implementation. Therefore this test module is
+named w.r.t. ivpfiltsmooth.py.
+"""
```

Comment:
the tests "the interface works" are separated from the tests "the numbers are good" now. both are written in pytest.

Reply:
That sounds like a very useful separation :)